### DISCUSSION TITLE
Security issue burn-down graph

### DISCUSSION THREAD

**Discussion by rossigee** [community/community#49472] [Code Security]
### Select Topic Area

Product Feedback

### Body

I've made some progress over the last few months persuading some of our organisation's repo owners to address some of their outstanding dependabot and other Security Alerts. We've halved the number of active issues, but still have a way to go.

It would be great feature to show some kind of graph showing the number of outstanding security issues per organisation/repo over time, so stakeholders can see how much progress is being made, or needs to be made, on staying on top of them.

**Comment by ghost** [comment]
That's a great idea! A burn-down graph showing the number of outstanding security issues per organization/repo over time would definitely be a helpful feature for stakeholders to track progress and prioritize which repos need more attention.

There are a few ways to approach implementing this. One option would be to use a third-party tool or service that integrates with GitHub and can generate these types of graphs. Another option would be to create a custom solution using GitHub's API and a graphing library like Chart.js.

Before diving into implementation, it might be helpful to gather feedback from stakeholders to understand their specific needs and what information they would like to see on the graph. Additionally, it's important to consider any security or privacy concerns that may arise from displaying this type of information publicly.
